requirements.txt

```diff
@@ -38,4 +38,5 @@ types-setuptools>=57.0.0
 types-requests
 types-setuptools>=57.0.0
 black==22.3.0
+cython-lint>=0.15.0; python_version >= "3.7"
```

Comment:
`cython-lint` is available only for Python 3.7+, and we run the `validate` step with 3.7.